ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -87,7 +89,7 @@ public MapExtract(Resource resource, IFileLoader fileLoader)
                 InitWorldExtract(resource);
                 break;
             default:
-                throw new InvalidDataException($"Resource type {resource.ResourceType} is not supported in {nameof(MapExtract)}.");
+                throw new InvalidDataException($"Resource type {resource.ResourceType} != supported in {nameof(MapExtract)}.");
```

Comment:
Incorrect replace

Reply:
why even replace this in the first place